src/schema.ts

```diff
@@ -273,44 +277,17 @@ export interface FlywheelConnectedEvent {
 }
 
 /**
- * Represents every channel that can be played on the vibraphone.
- *
- * We decided to start at one since the [[BassString]] type also
- * starts at one. We also decided to only represent playable
- * channels and not actual channels (two per note) because that is
- * an implementation detail of the physical MMX.
+ * All possible events relating to the machine itself.
  */
-export type VibraphoneChannel = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11;
```

Comment:
Not specified is the order of the channels. If reading from left to right, then 1 is the lowest pitch which is the opposite of the bass strings definition which says the 1 is the highest pitch.

Reply:
True, but should be left -> right not high low as the notes can be arranged arbitrarily.

Reply:
It's a little confusing to have it left -> right for the vibraphone but right -> left for the bass. But not a big deal.